packages/snap-store-mobx/src/Search/Stores/MerchandisingStore.ts

```diff
@@ -34,7 +34,8 @@ class Content extends Array {
 		return Array;
 	}
 
-	constructor(content) {
+	constructor(content: string[] | SearchResponseModelMerchandisingContentInline[] | undefined) {
+		// @ts-ignore
```

Comment:
Can we get rid of this @ts-ignore similar to how was done on other classes extending Array?